cmake/CMakeLists.txt

```diff
@@ -2,7 +2,15 @@
 # CMake build system
 # This file is part of LAMMPS
 # Created by Christoph Junghans and Richard Berger
-cmake_minimum_required(VERSION 2.8.12)
+if(PKG_USER-ADIOS)
+  message(STATUS "Force newer standards because of using ADIOS")
```

Comment:
Let's drop this message the other packages, which need a higher cmake version, e.g.`PKG_GPU` don't do that either.